tests/integration/test_generate_eq.py

```diff
@@ -220,3 +220,22 @@ def test_generate_eq_url_non_mandatory_event_date_is_returned(self):
 
         response = EqPayload()._find_event_date_by_tag('employment', collex_events_dates, '123', False)
         self.assertEqual(response, '2018-04-03')
+
+    @requests_mock.mock()
+    def test_generate_eq_payload_has_party_id(self, mock_request):
+
+        mock_request.get(url_get_case, json=case)
```

Comment:
This whole block of mock requests can be replaced with just these 4:

```python
mock_request.get(url_get_collection_exercise, json=collection_exercise)
mock_request.get(url_get_collection_exercise_events, json=collection_exercise_events)
mock_request.get(url_get_business_party, json=business_party)
mock_request.get(url_get_ci, json=collection_instrument_eq)
```
You shouldn't just copy a bunch of them from another test >_<  If you look at what the `create_payload` function is doing, you can see it's only doing calls to collection exercise, party and collection instrument, so you don't need the case mocks as they'll never be called.